src/main/charts/bitbucket/README.md

```diff
@@ -28,29 +28,23 @@ Kubernetes: `>=1.19.x-0`
 | affinity | object | `{}` | Standard Kubernetes affinities that will be applied to all Bitbucket pods Due to the performance requirements it is highly recommended running all Bitbucket pods in the same availability zone as your dedicated NFS server. To achieve this, you can define `affinity` and `podAffinity` rules that will place all pods into the same zone, and therefore minimise the real distance between the application pods and the shared storage. More specific documentation can be found in the official Affinity and Anti-affinity documentation:  https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity This is an example on how to ensure the pods are in the same zone as NFS server that is labeled with `role=nfs-server`:   podAffinity:    requiredDuringSchedulingIgnoredDuringExecution:      - labelSelector:          matchExpressions:            - key: role              operator: In              values:                - nfs-server # needs to be the same value as NFS server deployment        topologyKey: topology.kubernetes.io/zone |
 | bitbucket.additionalBundledPlugins | list | `[]` | Specifies a list of additional Bitbucket plugins that should be added to the  Bitbucket container. Note plugins installed via this method will appear as  bundled plugins rather than user plugins. An alternative to this method  is to install the plugins via "Manage Apps" in the product system  administration UI. These should be specified in the same manner as the  'additionalLibraries' property. |
 | bitbucket.additionalEnvironmentVariables | list | `[]` | Defines any additional environment variables to be passed to the Bitbucket  container. See https://hub.docker.com/r/atlassian/bitbucket-server for  supported variables. |
-| bitbucket.additionalJvmArgs | list | `["-XX:ActiveProcessorCount=2"]` |  system properties. |
```

Comment:
Am I right in saying that these properties are being removed with this PR, If so is that intended?

Reply:
This changes as Adam mentioned before is done by `pre-commit` and are not manual. What is happened is the list is removed as the items are documented and they look redundent

Reply:
I noticed the deleted documentation is introduced recently on https://github.com/atlassian-labs/data-center-helm-charts/pull/249/files by @tarka as part of DCKUBE-613: Configurable grace periods. 

We good to check on pre-commit configuration on Steve's machine to decide how to go on now. This point is raised as a comment in that PR by @badgersow as well. 
CC: @nanux